functions/package.json

```diff
@@ -8,7 +8,7 @@
     "start": "npm run shell",
     "deploy": "firebase deploy --only functions",
     "logs": "firebase functions:log",
-    "test": "jest"
+    "test": "jest --verbose"
```

Comment:
I might take this out just to not clog up logs for travis ci / etc. You can also pass args while running locally: e.g. `npm run test --verbose`

Reply:
Oh I did not know that! I'll change on the next commit!